policyengine_us/variables/gov/states/va/tax/income/deductions/itemized_deductions/va_prorate_fraction_head.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class va_prorate_fraction_head(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Virginia joint amount proration fraction of head"
```

Comment:
```suggestion
    label = "Federal adjusted gross income prorate fraction head"
```

Reply:
similar structure to ia_prorate_fraction.py but with net income instead of state net income